numpy/lib/function_base.py

```diff
@@ -4154,32 +4155,87 @@ def _median(a, axis=None, out=None, overwrite_input=False):
+            return False
+    return True
+
+
 def percentile(a, q, axis=None, out=None,
                overwrite_input=False, interpolation='linear', keepdims=False):
     """
     Compute the qth percentile of the data along the specified axis.
 
-    Returns the qth percentile(s) of the array elements.
+    The ``q``\ th percentile is synonymous with the `quantile` at ``q/100``.
+
+    Parameters
+    ----------
+    q : real number in range of [0,100] (or sequence of real numbers)
+        Percentile to compute, which must be between 0 and 100 inclusive.
+        This is divided by 100 and passed on to `quantile`.
+
+    All the other parameters have the same meaning as they do in `quantile`.
+
+    See also
+    --------
+    quantile
+    mean
+    median : equivalent to ``percentile(..., 50)``
+    nanpercentile
+
+    Examples
+    --------
+    >>> a = np.array([[10, 7, 4], [3, 2, 1]])
+    >>> a
+    array([[10,  7,  4],
+           [ 3,  2,  1]])
+    >>> np.percentile(a, 50)
+    3.5
+    >>> np.quantile(a, 0.5)
+    3.5
+
+
+    """
+    q = np.true_divide(q, 100)  # handles the asarray for us too
+    if not _valid_quantile(q):
+        raise ValueError("Percentiles must be in the range [0, 100]")
+    return quantile(a, q, axis, out, overwrite_input, interpolation, keepdims)
+
+
+def quantile(a, q, axis=None, out=None,
+             overwrite_input=False, interpolation='linear', keepdims=False):
+    """
+    Compute the `q`th quantile of the data along the specified axis.
+
+    ..versionadded:: 1.14
 
     Parameters
     ----------
     a : array_like
         Input array or object that can be converted to an array.
-    q : float in range of [0,100] (or sequence of floats)
-        Percentile to compute, which must be between 0 and 100 inclusive.
+    q : float in range of [0,1] (or sequence of floats)
+        Quantile to compute, which must be between 0.0 and 1.0 inclusive.
     axis : {int, sequence of int, None}, optional
-        Axis or axes along which the percentiles are computed. The
-        default is to compute the percentile(s) along a flattened
-        version of the array. A sequence of axes is supported since
-        version 1.9.0.
+        Axis or axes along which the quantiles are computed. The
+        default is to compute the quantile(s) along a flattened
+        version of the array.
+        .. versionchanged:: 1.9.0
+           A sequence of axes is supported
     out : ndarray, optional
         Alternative output array in which to place the result. It must
         have the same shape and buffer length as the expected output,
         but the type (of the output) will be cast if necessary.
     overwrite_input : bool, optional
         If True, then allow use of memory of input array `a`
         calculations. The input array will be modified by the call to
-        `percentile`. This will save memory when you do not need to
+        `quantile`. This will save memory when you do not need to
```

Comment:
I would just leave out the extended explanation, maybe just warn that the input array will be changed. This isn't the place for a tutorial, we just need a description of what the parameter does. I blame myself for insufficient review of the original.